person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    return true if @age.to_i >= 18
+
+    false
+  end
+
+  public
+
+  def can_use_services?
+    return true if @parent_permission == true && of_age?
+
+    false
+  end

```

Comment:
- [ ] This method should return true if person is of age **or** if they have permission from parents instead of if person is of age **and** if they have permission from parents.

- [ ] Kindly remove the public keyword before the `can_use_services?` method and move this method to come before the `of_age?` private method so that you do not need to add the public keyword to it because any ruby method without the private or protected keyword is assumed to be a public method.